massdriver.yaml

```diff
@@ -0,0 +1,108 @@
+schema: draft-07
+name: "azure-fhir-service"
+description: "Azure FHIR Service enables rapid exchange of health data using the Fast Healthcare Interoperability Resources (FHIR) data standard."
+source_url: github.com/massdriver-cloud/azure-fhir-service
+access: "private"
+type: "bundle"
+
+params:
+  examples: []
+  required:
+    - database
+    - registry
+    - logging
+  properties:
+    database:
+      title: Database
+      type: object
+      required:
+        - region
+      properties:
+        region:
```

Comment:
Simply for discussion. I've been putting `region` / `location` as a top-level param. I don't think we need to nest it here, what are your thoughts?